var/spack/repos/builtin/packages/icu4c/package.py

```diff
@@ -35,8 +43,8 @@ class Icu4c(AutotoolsPackage):
     configure_directory = 'source'
 
     def url_for_version(self, version):
-        url = "http://download.icu-project.org/files/icu4c/{0}/icu4c-{1}-src.tgz"
-        return url.format(version.dotted, version.underscored)
+        url = "https://github.com/unicode-org/icu/releases/download/release-{0}/icu4c-{1}-src.tgz"
+        return url.format(version.dashed, version.underscored)
```

Comment:
You can do something like:
```python
if version >= Version('65'):
    url = ...
    return url.format(...)
else:
    url = ...
    return url.format(...)
```
But aren't the older versions also available on GitHub? It looks like there's 507 releases on GitHub.

Reply:
Ah, I meant to consolidate those but then neglected to do so.

Reply:
> But aren't the older versions also available on GitHub? It looks like there's 507 releases on GitHub.

When I scrolled through the releases I did not see them. However, hitting the "next" button a few more times shows that they are there. The naming format changes though, and seems to be inconsistent with '-' and '_' in the names, but that was a quick look. It seems we could keep the naming somewhat consistent by maintaining use of the old download site, or have less naming consistency by using github and archives.

Reply:
I'll let you decide which you prefer.

Reply:
I will keep them pointing to the old download site.

Reply:
I just pushed a commit to clean up the version directives.